src/modules/Dashboard/reducers/dashboard.js

```diff
@@ -0,0 +1,66 @@
+import { handleActions } from "redux-actions";
+
+import {
+    DELETE_IMAGE_SUCCEEDED,
+    FETCH_USER_IMAGES_LIST_SUCCEEDED,
+    TOGGLE_NOTES_MODAL,
+    UPDATE_MODAL_VALUE,
+    UPDATE_NOTES_SUCCEEDED,
+} from "../constants/dashboard";
+
+const defaultState = {
+    imagesList: {},
+
+    imageId: null,
+    modalValue: null,
+    showNotesModal: false,
+    showSucceededMessage: false,
+};
+
+export default handleActions(
+    {
+        [DELETE_IMAGE_SUCCEEDED]: (state, { payload: imageId }) => {
+            const currentimagesList = { ...state.imagesList };
+            delete currentimagesList[imageId];
+            return ({
+                ...state,
+                imagesList: currentimagesList,
+            });
+        },
+        [FETCH_USER_IMAGES_LIST_SUCCEEDED]: (state, { payload: imagesList }) => {
+            const imagesObject = {};
+
+            imagesList.forEach((image) => {
+                const { image_id: imageId, ...rest } = image;
```

Comment:
You can not delete `image_id` from `image`. Then there will be no need to create an additional props [here](https://github.com/tolstoyevsky/cusdeb-web-client/pull/27/files#diff-abcb5639ad4993430ca26bb5d4879be8a77c078652cd610dc291dd5fa351830eR53).